.github/CODEOWNERS

```diff
@@ -14,3 +12,5 @@
 # Ensure that if any JSON files are added to the model directory, they are
 # still subject to full owners approval.
 /src/www/model/           @Jigsaw-Code/outline-owners
+/cordova-plugin-outline/  @Jigsaw-Code/outline-owners
+/src/www/cordova/         @Jigsaw-Code/outline-owners
```

Comment:
do you mean `/src/cordova/`? and how about `/src/electron/`?

Reply:
My bad. Fixed and added electron too.